roles/openshift_logging_mux/templates/mux.j2

```diff
@@ -66,6 +66,14 @@ spec:
         - name: muxcerts
           mountPath: /etc/fluent/muxkeys
           readOnly: true
+        - name: filebufferstorage
+{% if openshift_logging_mux_file_buffer_storage_type == 'pvc' %}
```

Comment:
openshift_logging_mux_file_buffer_mount_path is removed and the mount path "/var/log/mux/filebufferstorage" is fixed.  Thanks.